Description: replace cross-spawn by builtin child_process
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-12-24

--- a/test/cli.test.js
+++ b/test/cli.test.js
@@ -3,7 +3,7 @@
 let { test } = require('uvu')
 let { is, equal, match } = require('uvu/assert')
 let { join } = require('path')
-let spawn = require('cross-spawn')
+let { spawn } = require('child_process')
 
 let browserslist = require('..')
 let pkg = require('../package.json')
